README.md

```diff
@@ -1,2 +1,62 @@
 # inverted-index
 Inverted index data structure built with JavaScript
+
+[![Build Status](https://travis-ci.org/andela-korjiewuru/inverted-index.svg?branch=development)](https://travis-ci.org/andela-korjiewuru/inverted-index)
+[![Code Climate](https://codeclimate.com/github/andela-korjiewuru/inverted-index/badges/gpa.svg)](https://codeclimate.com/github/andela-korjiewuru/inverted-index)
+[![Coverage Status](https://coveralls.io/repos/github/andela-korjiewuru/inverted-index/badge.svg?branch=development)](https://coveralls.io/github/andela-korjiewuru/inverted-index?branch=development)
+
```

Comment:
Good! But your `readme` doesn't contain a section that introduces Inverted Index and/or what it is.